galaxychop/galaxychop.py

```diff
@@ -148,13 +143,21 @@ class Galaxy:
     pot_dark = np.loadtxt(path_potencial+'dark_ID_'+str(ID[j])+'.dat')
     pot_gas = np.loadtxt(path_potencial+'gas_ID_'+str(ID[j])+'.dat')
 
-    # Calculamos la energia.
-    E_tot_star = k_star - pot_star[:, 1]
-    E_tot_dark = k_dark - pot_dark[:, 1]
-    E_tot_gas = k_gas - pot_gas[:, 1]
+    def energy(self, pot_star, pot_dark, pot_gas):
+
+        '''Calculamos la energia cinetica y potencial y total'''
```

Comment:
In english please.

One liner docstring for functions or methods must be in the consecutive line, no linespaces after function signature.